### ISSUE TITLE
How to place the scrollbar always below the toolbar

### ISSUE THREAD

**Issue by mrksoftware** [vuematerial/vue-material#983]
Hi, 
I have a question for you (hope this is the right place):

How can I achieve a page layout like the one I see in all the [vuematerial.io](http://vuematerial.io/#/components/toolbar) website pages (in which the scrollbar is always below the fixed toolbar)?

![image](https://user-images.githubusercontent.com/7292153/29419205-c6a509dc-836e-11e7-9cce-85db0f7b5e80.png)

I've tried to download the source code of your website but I'm struggling to find the solution.

What I figured out so far is:

- The toolbar has a `position: relative;` CSS property, not fixed as I was expecting. But it manages anyway to stay on top even if the page scrolls.

- The div below the toolbar occupies all the remaining space without going over the visible area.
In this way, the scrollbar appears below the toolbar as I want. (I was not able to reproduce it in my project though) 

- If the div below the toolbar has a fixed height value which exceeds the visible area height, then a second scrollbar appears
    ![image](https://user-images.githubusercontent.com/7292153/29420062-3b242e76-8371-11e7-85e0-6b5649b5c5f4.png)

What I'm getting at the moment, is a scrollbar which occupies all the page:

![image](https://user-images.githubusercontent.com/7292153/29419313-1c8c8fd2-836f-11e7-9256-1b374ed6a25b.png)

The only way to achieve something like the [vuematerial.io](vuematerila.io) website is to set a fixed height but I want something more dynamic which works even if I resize the page.

Maybe is a dumb question but I've started using vue-material today, and I'm a little bit confused at the moment.

Thank you in advance for your time!

Mirko

**Comment by Samuell1** [comment]
I think best how to push scrollbar from top is adding padding to body

**Comment by mrksoftware** [comment]
Your trick works on the [vuematerial.io](http://vuematerial.io/#/components/toolbar) website, but not in the electron app I'm developing. 
The scrollbar is not affected by the padding. (I've tried both `body` and `html` selector with no success).

I've managed to find a solution, though:

Simply include this CSS rule under the div selector (I'm talking about the div below the toolbar, of course):

```css
height: calc(100vh - 64px);
```

In this way the height of that div is always equal to the 100% of the available viewport, minus 64px (which is the default height of the toolbar). 

Thank you for your support.

Mirko
